guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -40,21 +40,32 @@ For information on backups, see {AdministeringDocURL}Backing_Up_Server_and_Proxy
 
 ifdef::katello,satellite[]
 +
-. Regenerate certificates.
-On the main {Project} server:
+. Regenerate certificates on your {ProjectServer}:
+.. Regenerate certificates for {SmartProxies} that use default certificates:
 +
 [options="nowrap" subs="attributes"]
 ----
-# {certs-generate} --foreman-proxy-fqdn "myproxy.example.com" \
-                       --certs-update-all \
-                       --certs-tar "~/myproxy.example.com-certs.tar"
+# {certs-generate} --foreman-proxy-fqdn "{smartproxy-example-com}" \
+--certs-update-all \
+--certs-tar "~/{smartproxy-example-com}-certs.tar"
 ----
+.. Regenerate certificates for {SmartProxies} that use custom certificates:
 +
+[options="nowrap" subs="attributes"]
+----
+# {certs-generate} --foreman-proxy-fqdn "{smartproxy-example-com}" \
+--certs-tar "~/{smartproxy-example-com}-certs.tar" \
+--server-cert "/root/{certs-proxy-context}_cert/{certs-proxy-context}_cert.pem" \
+--server-key "/root/{certs-proxy-context}_cert/{certs-proxy-context}_cert_key.pem" \
+--server-ca-cert "/root/{certs-proxy-context}_cert/ca_cert_bundle.pem" \
+--certs-update-server
+----
 For more information on custom SSL certificates signed by a Certificate Authority, see {InstallingSmartProxyDocURL}deploying-a-custom-ssl-certificate-to-{smart-proxy-context}-server_{smart-proxy-context}[Deploying a Custom SSL Certificate to {SmartProxyServer}] in _{InstallingSmartProxyDocTitle}_.
 +
 endif::[]
 ifdef::katello[]
-. Copy the resulting tarball to your {SmartProxy}, for this example we will use `/root/myproxy.example.com-certs.tar`
+. Copy the resulting tarball to your {SmartProxy}.
+For this example, we will use `/root/myproxy.example.com-certs.tar`.
```

Comment:
IMHO this needs to be changed to the used attribute as below.

Reply:
Thanks for catching this! Applied :+1: